wbooks/src/app/screens/Login/index.tsx

```diff
@@ -17,13 +18,25 @@ interface DataForm {
 
 const Login = () => {
   const { control, handleSubmit, errors } = useForm();
-  const navigation = useNavigation();
   const dispatch = useDispatch();
 
-  const onSubmit = ({ email, password }: DataForm) => {
-    dispatch(actionCreators.login(email, password));
-    navigation.navigate('Library');
-  };
+  useEffect(() => {
```

Comment:
this should not be in login, this should be on the main screen to check if there is any loggedUser. Also all this actions should be defined on a service and then call them with a single function in the main scree of the app. And for main screen i mean the index that displays the whole app that has the provider, navigator, etc.

Reply:
Done!